website/server/libs/api-v3/pushNotifications.js

```diff
@@ -7,19 +7,23 @@ import Bluebird from 'bluebird';
 import {
   S3,
 } from './aws';
+import FCM from 'fcm-push';
 
-const GCM_API_KEY = nconf.get('PUSH_CONFIGS:GCM_SERVER_API_KEY');
+// const GCM_API_KEY = nconf.get('PUSH_CONFIGS:GCM_SERVER_API_KEY');
+const FCM_API_KEY = nconf.get('PUSH_CONFIGS:FCM_SERVER_API_KEY');
 
-let gcm = GCM_API_KEY ? pushNotify.gcm({
-  apiKey: GCM_API_KEY,
-  retries: 3,
-}) : undefined;
+let fcm = FCM_API_KEY ? new FCM(FCM_API_KEY) : undefined;
 
-if (gcm) {
-  gcm.on('transmissionError', (err, message, registrationId) => {
-    logger.error('GCM Error', err, message, registrationId);
-  });
-}
+// let gcm = GCM_API_KEY ? pushNotify.gcm({
```

Comment:
@paglias mentioned above that the push notify library supports FCM but doesn't have the update version of node-gcm. So, I figured if we would leave this code incase they update. I also suggested removing it - so either way is fine with me.